packages/importer/src/huly/cards.ts

```diff
@@ -81,20 +81,22 @@ export class CardsProcessor {
     await this.processMetadata(directoryPath, result, topLevelTypes)
 
     const typesRefs = topLevelTypes.map((type) => type.props._id) as Ref<MasterTag>[]
-    const updateDefaultSpace: UnifiedUpdate<CardSpace> = {
-      _class: card.class.CardSpace,
-      _id: 'card:space:Default' as Ref<CardSpace>,
-      space: core.space.Model,
-      props: {
-        $push: {
-          types: {
-            $each: [...new Set(typesRefs)],
-            $position: 0
+    if (typesRefs.length > 0) {
+      const updateDefaultSpace: UnifiedUpdate<CardSpace> = {
+        _class: card.class.CardSpace,
+        _id: 'card:space:Default' as Ref<CardSpace>,
+        space: core.space.Model,
+        props: {
+          $push: {
+            types: {
+              $each: [...new Set(typesRefs)],
```

Comment:
Why we need $each, instead of `$push: { types: [...new Set(typesRefs)] }`

Reply:
![image](https://github.com/user-attachments/assets/0280c06d-b7a9-40c2-8c1d-7000a323a17b)

Actually can't do that - the type system requires `$each` here. As you can see from the error, direct array assignment isn't compatible with the expected type `Ref<MasterTag> | Position<Ref<MasterTag>> | undefined`. We'd need to modify core types to make it work:

The `DocumentUpdate` type uses `PushOptions` which expects either a single value or a `Position` object with `$each` and `$position` for array operations. This is part of our core types in `tx.ts`, 

For now, I think it's safer to stick with the current implementation. Let me know if you'd like to explore changing the base types though.